okta/resource_okta_policy_password.go

```diff
@@ -126,6 +126,14 @@ func resourcePolicyPassword() *schema.Resource {
 				Description: "If a user should be informed when their account is locked.",
 				Default:     false,
 			},
+			"lockout_notifications": {
```

Comment:
Can you name this `lockout_notification_channels`?

Reply:
Thanks a lot! I will commit the changes soon, appreciate the patience of going through the mismatch of data-types , conversions and all silly mistakes I made :)

Also, I have raised the following PR to extend the policies template:

https://github.com/articulate/oktasdk-go/pull/36

Reply:
Hi @quantumew . Let me know if the sdk change should be done and if you need me to adjust something for this option.

Cheers!